<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Group Accounts</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LM Roman 6</family>
     <pointsize>16</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>GROUP ACCOUNTS AND TRANSACTIONS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SETTINGS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>18</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>ACCOUNTS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_acc_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>ADD NEW ACCOUNT</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>691</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>141</number>
   </attribute>
   <column>
    <property name="text">
     <string>Account Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Account Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date Created</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Balance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>16</width>
     <height>681</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Transactions History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>120</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Generate Full Report</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>150</y>
     <width>651</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>108</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Account ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_accs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Total Accounts Balance:   KSH. xxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_active">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Total Active Accounts: X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Treasurer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tresh_edit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>620</y>
     <width>491</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
